subprojects/core/src/test/groovy/org/gradle/process/internal/DefaultProcessForkOptionsTest.groovy

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2010 the original author or authors.
+ * Copyright 2024 the original author or authors.
```

Comment:
💅

Reply:
I reverted it. What is the policy here? I see that IntelliJ automatically updates that